apps/src/templates/teacherDashboard/SectionTable.jsx

```diff
@@ -154,7 +158,7 @@ class SectionTable extends Component {
     onEdit: PropTypes.func,
 
     //Provided by redux
-    sectionRows: PropTypes.arrayOf(sectionDataPropType),
+    sectionRows: PropTypes.arrayOf(sortableSectionShape).isRequired,
```

Comment:
looks like this is missing from the imports in this file

Reply:
https://github.com/code-dot-org/code-dot-org/pull/17659/files/eedd6b11be46cd2d6facd95aacaea77ecd1e9850#diff-bc2049077fe3623e62caf349ff548e6dR11

Reply:
ah sorry, for some reason I thought I was at the top of the imports in the file